pmlc/conversion/tile_to_pxa/pass.cc

```diff
@@ -234,39 +304,51 @@ struct FirstOperand {
   }
 };
 
+void promoteTypes(ArrayRef<Value> operands, llvm::SmallVectorImpl<Value>* into) {
+  // TODO: add cast op for each operand wherever appropriate
```

Comment:
I presume this code is meant to eventually compute the common subtype + then cast...

Reply:
Yep, will do in a follow up